analyzers/tests/SonarAnalyzer.UnitTest/Rules/AsyncVoidMethodTest.cs

```diff
@@ -28,51 +28,59 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class AsyncVoidMethodTest
     {
+        private readonly VerifierBuilder builder = new VerifierBuilder<AsyncVoidMethod>();
+
         [TestMethod]
         public void AsyncVoidMethod() =>
-            OldVerifier.VerifyAnalyzer(@"TestCases\AsyncVoidMethod.cs", new AsyncVoidMethod());
+            builder.AddPaths("AsyncVoidMethod.cs")
+                .Verify();
 
 #if NET
         [TestMethod]
         public void AsyncVoidMethod_CSharp9() =>
-            OldVerifier.VerifyAnalyzerFromCSharp9Library(@"TestCases\AsyncVoidMethod.CSharp9.cs", new AsyncVoidMethod());
+            builder.AddPaths("AsyncVoidMethod.CSharp9.cs")
+                .WithOptions(ParseOptionsHelper.FromCSharp9)
+                .Verify();
 
         [TestMethod]
         public void AsyncVoidMethod_CSharpPreview() =>
-            OldVerifier.VerifyAnalyzerCSharpPreviewLibrary(@"TestCases\AsyncVoidMethod.CSharpPreview.cs", new AsyncVoidMethod());
+            builder.AddPaths("AsyncVoidMethod.CSharpPreview.cs")
+                .WithLanguageVersion(Microsoft.CodeAnalysis.CSharp.LanguageVersion.Preview)
+                .Verify();
 
         [TestMethod]
         public void AsyncVoidMethod_CSharp10() =>
-            OldVerifier.VerifyAnalyzerFromCSharp10Library(
-                @"TestCases\AsyncVoidMethod.CSharp10.cs",
-                new AsyncVoidMethod(),
-                NuGetMetadataReference.MicrosoftVisualStudioQualityToolsUnitTestFramework);
+            builder.AddPaths("AsyncVoidMethod.CSharp10.cs")
+                .WithOptions(ParseOptionsHelper.FromCSharp10)
+                .AddReferences(NuGetMetadataReference.MicrosoftVisualStudioQualityToolsUnitTestFramework)
+                .Verify();
 
         [DataTestMethod]
         [DataRow("1.1.11")]
         [DataRow(Constants.NuGetLatestVersion)]
         public void AsyncVoidMethod_MsTestV2_CSharpPreview(string testFwkVersion) =>
-            OldVerifier.VerifyAnalyzerCSharpPreviewLibrary(
+            builder.AddPaths(
```

Comment:
The line ordering is confusing to read. Let's put this comment before the `WithLanguageVersion` instead so we can have `AddPaths` as a single line